modules/remotecontrol/index.js

```diff
@@ -204,18 +213,31 @@ class RemoteControl {
 
     /**
      * Sends remote control event to the controlled participant.
+     *
      * @param {Object} event the remote control event.
      */
     sendEvent(event) {
         const remoteControlEvent = Object.assign(
-            { name: REMOTE_CONTROL_EVENT_NAME },
+            { name: REMOTE_CONTROL_MESSAGE_NAME },
             event
         );
         this.channel.send({
             method: 'message',
             params: { data: remoteControlEvent }
         });
     }
+
+    /**
+     * Sends a response of a request from Jitsi Meet.
+     *
+     * @param {Object} response the response.
```

Comment:
You are totally right! When I looked how to change it, I realized that the method actually only sends a message :) It can be easily used to send any message not only responses! So I renamed the method. I hope the new version is clearer!